src/components/InfographicsSection.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import '../styles/InfographicsSection.scss'
+import Icon from "./Icon";
+import {EIconName} from "../enums/EIconName";
+
+interface IProps {
+    id: number,
+    isActive: boolean,
+    handleSelectSection: (id: number) => void
+}
+
+const InfographicsSection: React.FC<IProps> = (props) => {
+
+    const {id, isActive, handleSelectSection} = props;
+
+    const onSelectSection = () => {
+        handleSelectSection(id);
+    };
+
+    return (
+        <div className="scope__InfographicsSection">
+            <div
+                className={`infographics__infographics-section ${isActive && 'infographics__infographics-section--active'}`}
```

Comment:
je lepsi pouzit

`${isActive ? 'infographics__infographics-section--active' : ''}`

protoze kdyz tam mas `&&` tak ti to do HTML dava tridu `false` :)